src/openrct2/platform/Platform.Linux.cpp

```diff
@@ -97,25 +97,23 @@ namespace Platform
         }
         // 2. Try {${exeDir},${cwd},/}/{data,standard system app directories}
         // exeDir should come first to allow installing into build dir
-        std::vector<std::string> prefixes;
-        auto exePath = Platform::GetCurrentExecutablePath();
-        prefixes.push_back(Path::GetDirectory(exePath));
-        prefixes.push_back(GetCurrentWorkingDirectory());
-        prefixes.push_back("/");
-        static const char* SearchLocations[] = {
+        // clang-format off
+        const std::string prefixes[]{
+            Path::GetDirectory(Platform::GetCurrentExecutablePath()),
+            GetCurrentWorkingDirectory(),
+            "/"
+        };
+        static constexpr u8string_view SearchLocations[] = {
             "/data",
             "../share/openrct2",
-#    ifdef ORCT2_RESOURCE_DIR
```

Comment:
I've searched the whole codebase, this seems to be a dead preprocessor variable (not found in CMake - in any of them and no easy way to actually inject it - yes it's doable, but should we?)

this commit seems to have removed it:
 1f8f354e49b0fa47bd8d1c1e7bca633809acec06
Author: Ted John <ted@brambles.org>
Date:   Mon May 8 17:35:15 2017 +0100

    Update root cmake and enable tests

Reply:
Maybe @janisozaur has some idea about what should be here.

Reply:
This has been defunct for so much time I'm ok with dropping this.